action.yml

```diff
@@ -18,6 +18,7 @@ inputs:
   include-prerelease:
     description: 'Whether prerelease versions should be matched with non-exact versions (for example 5.0.0-preview.6 being matched by 5, 5.0, 5.x or 5.0.x). Defaults to false if not provided.'
     required: False
+    default: 'false'
```

Comment:
Could you please remove single quotes ?

Reply:
@dmitry-shibanov 
GitHub Action schema warns this should be string. Should I still remove the quotes?
![Incorrect type. Expected "string".](https://user-images.githubusercontent.com/24802730/182259566-1fae793c-8a13-4eea-b8e3-cef4972cc437.png)

Reply:
I think it should be okay, because we use the similar approach for setup-node https://github.com/actions/setup-node/blob/main/action.yml#L16. But I think you can keep this variant due to the parsing logic of runner.